cci/CMakeLists.txt

```diff
@@ -109,13 +112,21 @@ else(VERSION_MATCHES_LENGTH GREATER 3)
       message(FATAL_ERROR "Could not get count information from Git")
     endif(git_result)
     set(CCI_EXTRA_VERSION ${commit_count})
+    execute_process(COMMAND ${GIT_EXECUTABLE} rev-parse --short=7 HEAD
+      OUTPUT_VARIABLE commit_hash RESULT_VARIABLE git_result
+      ERROR_QUIET OUTPUT_STRIP_TRAILING_WHITESPACE
+      WORKING_DIRECTORY ${CCI_ROOT_PATH})
+    if(git_result)
+      message(FATAL_ERROR "Could not get hash information from Git")
+    endif(git_result)
+    set(CCI_HASH_TAG -${commit_hash})
     # Generate the same file in other directory to trigger cmake re-configure when the HEAD changes
     # configure_file("${CCI_ROOT_PATH}/.git/HEAD" ${CMAKE_CURRENT_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/HEAD.check_cache COPYONLY)
   else(EXISTS "${CCI_ROOT_PATH}/.git" AND GIT_FOUND)
     set(CUBRID_EXTRA_VERSION "0000")
   endif(EXISTS "${CCI_ROOT_PATH}/.git" AND GIT_FOUND)
 endif(VERSION_MATCHES_LENGTH GREATER 3)
-set(CCI_BUILD_NUMBER ${CCI_MAJOR_VERSION}.${CCI_MINOR_VERSION}.${CCI_PATCH_VERSION}.${CCI_EXTRA_VERSION})
+set(CCI_BUILD_NUMBER ${CCI_MAJOR_VERSION}.${CCI_MINOR_VERSION}.${CCI_PATCH_VERSION}.${CCI_EXTRA_VERSION}${CCI_HASH_TAG})
```

Comment:
And,
set(CCI_BUILD_NUMBER ${CCI_MAJOR_VERSION}.${CCI_MINOR_VERSION}.${CCI_PATCH_VERSION}.${CCI_EXTRA_VERSION}-${CCI_HASH_TAG})